python/feathub/feature_views/feature_view.py

```diff
@@ -169,3 +170,18 @@ def _get_keys(self) -> Optional[List[str]]:
                 key_fields.extend(keys)
 
         return list(OrderedDict.fromkeys(key_fields))
+
+    def is_bounded(self) -> bool:
+        return cast(TableDescriptor, self.source).is_bounded()
+
+    def get_bounded_view(self) -> TableDescriptor:
+        if self.is_bounded():
+            return self
+
+        if self.is_unresolved():
```

Comment:
It seems simpler to remove this since `self.is_bounded()` should have thrown an exception if the table is unsolved.